pyartcd/pyartcd/pipelines/ocp4.py

```diff
@@ -777,10 +777,17 @@ async def _sync_images(self):
 
         records = record_log.get('build', [])
         operator_nvrs = []
-
+        operator_nvrs_map = {}
+        operator_deps = []
         for record in records:
             if record['has_olm_bundle'] == '1' and record['status'] == '0' and record.get('nvrs', None):
                 operator_nvrs.append(record['nvrs'].split(',')[0])
+            if record['status'] != '0' and record.get('nvrs', None) and record.get('dependents', None):
+                operator_deps.extend([dep for dep in record['dependents'].split(',') if dep not in operator_deps])
```

Comment:
use a set, it will deduplicate:
`depenents.update(record['dependents'].split(','))`

Reply:
also we are storing failed dependents, so the name should reflect that `failed_dependents`

Reply:
We are storing failed_images_dependents so the name should reflect that ..